### DISCUSSION TITLE
Issues with Sharepoint Server 2019

### DISCUSSION THREAD

**Discussion by mmschmidtarc** [microsoft-search/pnp-modern-search#2822] [Q&A] [answered]
We are trying to install PNP Modern Search V4 to Sharepoint 2019 Server (on-premise), but keep getting an error message.  Does this work with an on-prem instance, or is this only for the cloud?  If so, is there a version that will work for an on-premise Sharepoint 2019 version?

**Comment by kasperbolarsen** [comment, marked as answer]
Hi, please see https://github.com/pnp/sp-dev-solutions/issues/355#issuecomment-725252440

**Reply by mmschmidtarc** [reply]
Thanks! That worked.
